app/webdav/server/methods/getFileFromWebdav.js

```diff
@@ -17,13 +17,21 @@ Meteor.methods({
 		if (!account) {
 			throw new Meteor.Error('error-invalid-account', 'Invalid WebDAV Account', { method: 'addNewWebdavAccount' });
 		}
-		const client = createClient(
-			account.server_url,
-			{
-				username: account.username,
-				password: account.password,
-			}
-		);
+
+		const client = account.token
```

Comment:
this block of code is duplicated across three files.. please create a function to do this and use it on instead of duplicating code.